igvc_ws/src/sensors/src/gps_node.py

```diff
@@ -1,30 +1,35 @@
 #!/usr/bin/env python
 
+'''
+    Discription: GPS ROS node, keeps track of the last known GPS location
+'''
+
 import ros_api as ros
 from ros_api import println
 import custom_msgs.msg as msgs
+import rospy
```

Comment:
Again, ropsy not needed, ros.spin() works for line 32 (you can check the wiki reference I created.